vllm/spec_decode/batch_expansion.py

```diff
@@ -314,14 +314,9 @@ def _create_target_seq_group_metadata(
         # this also controls whether the probs get modified in the sampler
         # (see use of _modify_greedy_probs_inplace there).
```

Comment:
This block of comments can also be removed.

Reply:
Thank you for reviewing! I removed the intermediate variable and blocks of comments.